src/briefcase/platforms/windows/__init__.py

```diff
@@ -82,15 +82,30 @@ def run_app(self, app: BaseConfig, **kwargs):
         self.logger.info("Starting app...", prefix=app.app_name)
         try:
             # Start streaming logs for the app.
-            self.logger.info("=" * 75)
-            self.tools.subprocess.run(
+            log_popen = self.tools.subprocess.Popen(
                 [os.fsdecode(self.binary_path(app))],
                 cwd=self.tools.home_path,
-                check=True,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.STDOUT,
+                bufsize=1,
             )
+
+            try:
+                # Start streaming logs for the app.
+                self.logger.info("=" * 75)
+                self.tools.subprocess.stream_output(
+                    app.app_name,
+                    log_popen,
+                )
+            finally:
+                self.tools.subprocess.cleanup(app.app_name, log_popen)
+
+            # If the process didn't exit cleanly, raise an error.
+            if log_popen.returncode != 0:
+                raise BriefcaseCommandError(f"Problem running app {app.app_name}.")
         except KeyboardInterrupt:
             pass  # Catch CTRL-C to exit normally
-        except subprocess.CalledProcessError as e:
+        except OSError as e:
             raise BriefcaseCommandError(f"Unable to start app {app.app_name}.") from e
```

Comment:
I was wondering this about the Linux updates as well....why is `subprocess.cleanup()` being called here? The call to `stream_output()` will handle this, right? or is this for particular corner cases?

Reply:
Nope - it's oversight. I was copying the general pattern from macOS where there *is* a cleanup, but forgot that because we're streaming the actual process we care about, the cleanup is done automatically by `stream_output()`.